app/views/manage_assessments/assignments/new.html.erb

```diff
@@ -1,5 +1,45 @@
-<%= simple_form_for @assignment, url: manage_assessments_outcome_coverage_assignments_path do |form| %>
-  <%= form.input :name, label: t(".assignment") %>
-  <%= form.input :problem, label: t(".problem") %>
+<%= content_for :full_bleed do %>
```

Comment:
Should we turn this into a partial since we use it here and on the add a class/outcome form?